pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/PbjCompilerTask.java

```diff
@@ -35,6 +35,8 @@
 
 /** Gradle Task that generates java src code from protobuf proto schema files. */
 public abstract class PbjCompilerTask extends SourceTask {
+    private final static int MAX_TRACE_FRAMES = 8;
+    private static final String STACK_ELEMENT_INDENT = "    ";
```

Comment:
Nit: please list type modifiers in a consistent order, e.g. always `static final`.

Reply:
Missed that, I think the IDE mixed one of them.
Thanks, and done.